installer/pkg/components/node-exporter/daemonset.go

```diff
@@ -55,11 +55,7 @@ func daemonset(ctx *common.RenderContext) ([]runtime.Object, error) {
 							RunAsUser:    pointer.Int64(65534),
 						},
 						ServiceAccountName: Name,
-						Tolerations: []v1.Toleration{
-							{
-								Operator: v1.TolerationOpExists,
-							},
-						},
+						Tolerations:        ctx.Config.Tolerations,
```

Comment:
I'm not entirely familiar with tolerations... How would node-exporter behave with this change? Would it get deployed only to the nodes for the monitoring nodepool?

If yes, that's not very useful since we would have node metrics where gitpod components or workspaces are running

Reply:
:facepalm: You are right. Fixed.